gen.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import shutil
+import subprocess
+
+from jinja2 import Environment, FileSystemLoader
+
+SRC_PATH = 'src'
+WORKPLACE_PATH = 'workplace'
+SCHEMA_FILENAME = 'events.json'
+GEN_PY_FILENAME = 'events.py'
+TEMPLATE_PATH = 'templates'
+INIT_PY_TEMPLATE = 'init_py.template'
+PACKAGE_ROOT = 'google/events'
+
+jinja2_env = Environment(loader=FileSystemLoader(searchpath=TEMPLATE_PATH))
```

Comment:
Can we have a file comment as to what this file is doing?

Reply:
Docstring added.